<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StandBrowserToolboxWidget</class>
 <widget class="QDialog" name="StandBrowserToolboxWidget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StandBrowser Toolbox</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QTabWidget" name="tb">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabGrid">
    <attribute name="title">
     <string>Grid</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>491</width>
       <height>191</height>
      </rect>
     </property>
     <property name="plainText">
      <string>DESCRIPTION

1) Select layer from drop down.

2) Select distribution algorithm (currently only random distribution)

3) Choose if creating new layer from template or adding to existing. The template is made with field name and types from existing file, but geometries as points (currently only creating new from template).

4) Select 'Apply or 'Close'.</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbLayer">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>210</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Use selected stand from layer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>221</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select distribution algorithm:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cbAlgo">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>240</y>
       <width>131</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>280</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of points/stand:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbMinPoint">
     <property name="geometry">
      <rect>
       <x>351</x>
       <y>280</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="sbMaxPoint">
     <property name="geometry">
      <rect>
       <x>441</x>
       <y>280</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>20</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>280</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>280</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Min</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbTemplate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>381</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string> Create new layer from template</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbTemplate">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>340</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbExisting">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>381</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Add to existing layer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="leTemplate">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>410</y>
       <width>351</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPrint">
    <attribute name="title">
     <string>Print</string>
    </attribute>
    <widget class="QTextEdit" name="teOutput">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="bbDialog">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>490</y>
     <width>501</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Close</set>
   </property>
  </widget>
  <zorder>bbDialog</zorder>
  <zorder>tb</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
